sk0-front/src/app/auth/register/register.page.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+import { IonicModule } from '@ionic/angular';
+import { AlertController } from '@ionic/angular';
+
+@Component({
+  selector: 'app-register',
+  templateUrl: './register.page.html',
+  styleUrls: ['./register.page.scss'],
+  standalone: true,
+  imports: [IonicModule, CommonModule, FormsModule]
+})
+export class RegisterPage {
+  name: string = '';
+  email: string = '';
+  password: string = '';
+
```

Comment:
Nem lehet, hogy egyszerűbb lenne egy FormGroup-ot létrehozni és abba legyen a name/email/password külön-külön FormControlként? Úgy szerintem a validálás is könnyebb a beépített cuccok miatt, de lehet csak személyes preferencia kérdése :)

Reply:
olyat még nem használtam eddig, de beszéljük meg, jól hangzik